zebrad/Cargo.toml

```
@@ -55,6 +55,7 @@ once_cell = "1.8"
 regex = "1.4.6"
 semver = "1.0.3"
 tempdir = "0.3.7"
+tokio = { version = "0.3.6", features = ["time", "rt-multi-thread", "stream", "macros", "tracing", "signal", "test-util"] }
```

Nit: We try to use the full features in dev dependencies, it's a bit simpler.

```suggestion
tokio = { version = "0.3.6", features = ["full"] }
```